SuperScouting/src/App.js

```diff
@@ -37,7 +54,7 @@ function csvStringify(data) {
 class App extends React.Component {
     constructor(props) {
         super(props);
-        this.state = { signedIn: false, ScouterName: "", EventName: "", selected: 'sign-in', fouls: [], matchSchedule: null, team1: options[0], team2: options[0], team3: options[0], matchNumber: null };
+        this.state = { signedIn: false, ScouterName: "", EventName: "", selected: 'sign-in', fouls: [], matchSchedule: null, team1: options[0], team2: options[0], team3: options[0], matchNumber: null, scouterColor: null };
```

Comment:
There's already a state called `Alliance` which stores the color (0 = red, 1 = blue). I forgot to add it to the constructor but it does get set (see line 81). Therefore I don't think you need any of the code dealing with `scouterColor`, you can just use `Alliance`